frontend/src/main/java/de/iav/frontend/controller/TimeSlotsController.java

```diff
@@ -100,4 +101,16 @@ private void noTimeSlotIsSelectedMessageBox(String contentText){
             }
         });
     }
+
+    public void onUsersDataButtonClick(ActionEvent event) throws IOException {
+        sceneSwitchService.switchToUsersController(event, loggedUser);
+    }
+
+    public void onLogoutButtonClick(ActionEvent event) throws IOException {
+        if(authService.logout())
+        {sceneSwitchService.switchToLoginController(event);}
```

Comment:
Für die ganze Datei (am besten in allen Dateien immer wieder) cmd+opt+l = code formatieren

Reply:
Umgesetzt